default.hbs

```diff
@@ -22,14 +22,12 @@
 
     {{ghost_foot}}
 
-    <script src={{asset "built/js/vendor/swiper/swiper.js"}}></script>
```

Comment:
This change should be standalone - so don't remove swiper (until we add it back somewhere else)